chassis/utils/chassis_module_set_admin_state.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+
+# Name: chassis_module_set_admin_state.py, version: 1.0
+# Syntax: chassis_module_set_admin_state.py <module_name> <up/down>
+#
+import re
+import sys
+import subprocess
+import time
+
+from platform_ndk import nokia_common
+from sonic_py_common.logger import Logger
+
+logger=Logger("chassis_module_set_admin_state.py")
+
+sfm_asic_dict = {
+    0: [0,1],
+    1: [2,3],
+    2: [4,5],
+    3: [6,7],
+    4: [8,9],
+    5: [10,11],
+    6: [12,13],
+    7: [14,15]
+}
+
+sfm_hw_slot_mapping = {
+    0: 15,
+    1: 16,
+    2: 17,
+    3: 18,
+    4: 19,
+    5: 20,
+    6: 21,
+    7: 22
+}
+
+def main():
+    argc = len(sys.argv)
+
+    if argc != 3:
+        logger.log_warning("Missing parameters!")
+        return
+
+    module = sys.argv[1]
+    state = sys.argv[2]
+
+    if not module.startswith("FABRIC-CARD"):
+        logger.log_warning("Failed to set {} state. Admin state can only be set on Fabric module".format(module))
+        return
+
+    num = int(re.search(r"(\d+)$", module).group())
+    if num not in sfm_hw_slot_mapping:
+        logger.log_error("Invalid module name {}".format(module))
+        return
+
+    if state == "down":
+        logger.log_info("Shutting down chassis module {}".format(module))
```

Comment:
Use "shutdown" instead of "shuting down"

Reply:
Shutdown is a noun. A verb should be here.
<img width="377" alt="image" src="https://github.com/nokia/sonic-platform/assets/134556118/51f76288-6665-4ecc-b8c5-95caa30b28e4">

https://dictionary.cambridge.org/us/dictionary/english/shutdown?q=Shutdown#google_vignette

Reply:
Addressed.